src/components/CreateAdministration.vue

```diff
@@ -24,10 +24,10 @@
               <PvInputText
                 id="administration-public-name"
                 v-model="state.administrationPublicName"
-                style="width: 20vh"
+                style="width: 50vh"
```

Comment:
I'd rather not specify the width here and instead use the `formgrid` class to make things more responsive.
```suggestion
```

Can you combine the `administration-name`, `administration-public-name`, and `dates` input using something like `class="field col-12 md:col-6"`? See https://primeflex.org/formlayout#advanced for a more complete example. I think for sufficiently wide screens, the three should all be on the same line. But as the screen gets narrower, they should each be on their own line.